src/app/components/homepage/cards-container/card-content-detail/card-content-detail.component.ts

```diff
@@ -7,12 +15,50 @@ import { Component, OnInit, OnDestroy, Output } from '@angular/core';
 })
 export class CardContentDetailComponent implements OnInit, OnDestroy {
     title: string;
+    postId: string;
     description: string;
     url: string;
     type: string;
+    counterLike: number;
+    counterDisl: number;
+    private userSub: Subscription;
+    private routeSubscription: Subscription;
+    media: Media = new Object();
+    postIdroute: string;
 
-    constructor() {}
+    constructor(
+        private store: Store<fromApp.AppState>,
+        private getDataService: GetDataService,
+        private route: ActivatedRoute,
+    ) {
+        this.routeSubscription = this.route.params.subscribe(
+            params => (this.postIdroute = params.postId),
+        );
+    }
 
-    ngOnInit(): void {}
-    ngOnDestroy() {}
+    ngOnInit(): void {
+        this.userSub = this.store.select('likesCount').subscribe(like => {
+            console.log('PostId detail from store', like.postId);
+            this.counterLike = +like.likes;
+            this.counterDisl = +like.dislikes;
+        });
+        this.render(this.postIdroute);
+    }
+    onLike() {
+        this.store.dispatch(new LikesounterActions.LikesLike());
+        console.log('Get post id', this.postIdroute);
+    }
+
+    onDisLike() {
+        this.store.dispatch(new LikesounterActions.LikeDislike());
```

Comment:
It is strange that you don't take into account who liked and who disliked

Reply:
It will be in next subtask. Store is not contain full logic now)

Reply:
ok